apps/challenges/migrations/0055_added_leaderboard_order_is_descending_field.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
```

Comment:
Please change the file name to - `add_leaderboard_order_is_descending_field_in_phase_model`

Reply:
@RishabhJain2018 done